src/class/any_object.rs

```diff
@@ -93,6 +94,24 @@ impl AnyObject {
     pub fn to<T: Object>(&self) -> T {
         T::from(self.value)
     }
+
+    /// Determines the value type of the object
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use ruru::{AnyObject, Fixnum, VM};
+    /// use ruru::traits::Object;
+    /// use ruru::types::ValueType;
+    /// # VM::init();
+    ///
+    /// let any_object = Fixnum::new(1).to_any_object();
+    ///
+    /// assert_eq!(any_object.get_type(), ValueType::Fixnum);
+    /// ```
+    pub fn get_type(&self) -> ValueType {
```

Comment:
Right. For some reason I think I got a compile error when I named it `type`. I'll try again.